### PR TITLE
Make the bulk upload data json file proper json

### PR DESCRIPTION
**Pull request by trane2**
Finished part 1 of issue #3123. Bulk upload data json more readable now.

### CODE REVIEW

**Comment by bmcutler** [comment]
Please resolve conflicts

**Comment by bmcutler** [comment]
please resolve conflicts

File: site/app/controllers/student/SubmissionController.php [hunk 1]
```diff
@@ -786,13 +786,17 @@ private function ajaxUploadSplitItem() {
...
+                                  "upload_timestamp" =>  $upload_time_string_tz,
+                                  "filepath" => $uploaded_file);
+        
+        $bulk_upload_data_json = json_encode($bulk_upload_data, JSON_PRETTY_PRINT);
         
-        if (!@file_put_contents(FileUtils::joinPaths($version_path, "bulk_upload_data.json"), serialize($bulk_upload_data_json)."\n")) {
+        if (!@file_put_contents(FileUtils::joinPaths($version_path, "bulk_upload_data.json"), $bulk_upload_data_json."\n")) { 
+
             return $this->uploadResult("Failed to create bulk upload file for this submission.", false);
         }
```
Comment (MasterOdin) [inline site/app/controllers/student/SubmissionController.php]: Use `FileUtils:;writeJsonFile` for writing PHP arrays to JSON files. There's not a ton of point to adding the `\n` to the end of the file.
```
        $bulk_upload_data = [
            "submit_timestamp" =>  $current_time_string_tz,
            "upload_timestamp" =>  $upload_time_string_tz,
            "filepath" => $uploaded_file
        ];

        if (FileUtils::writeJsonFile(FileUtils::joinPaths($version_path, "bulk_upload_data.json"), $bulk_upload_data)) {
            return $this->uploadResult("Failed to create bulk upload file for this submission.", false);
        }
```

**Comment by bmcutler** [comment]
@trane2 please address comments

**Comment by bmcutler** [comment]
@trane2 @MasterOdin 
Did not yet address all comments?

**Pull request merged**